server_action_mass_edit/wizard/mass_editing_wizard.py

```diff
@@ -281,9 +299,44 @@ def create(self, vals_list):
                             values.update({split_key: m2m_list})
 
                 if values:
```

Comment:
Maybe you can avoid the complexity warning `C901` like this
```suggestion
                if not values:
                    continue
```

Reply:
Changed, but `noqa: C901` already is needed

Reply:
@chienandalu is OK?